src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/common/ApplicationLogger.java

```diff
@@ -54,10 +54,6 @@ public void info(String message) {
         logger.info(message, null);
     }
 
-    public void info(String message, Map<String, Object> map) {
```

Comment:
I don't think this method needs to be removed does it?

Reply:
it wasn't being called by anything, the only thing to use it was two lines of logging that are completely removed for being all sensitive data. I can add it back in but there's no real point

Reply:
It's a tricky one. It's not used so should it be there or the class is a wrapper class basically for structured logging calls so it might be handy to leave it there

Reply:
I'll add it back in to be safe 👍